### ISSUE TITLE
Don't move .htaccess out of the site into root

### ISSUE THREAD

**Issue by fluxsauce** [aquifer/aquifer#55]
``` bash
jon@Decker ~/projects/REMOVED (master) $ ls -la root
total 0
drwxr-xr-x   3 jon  staff   102 Sep 29 14:28 .
drwxr-xr-x  38 jon  staff  1292 Sep 29 14:02 ..
-rw-r--r--   1 jon  staff     0 Sep 29 12:23 .gitkeep
jon@Decker ~/projects/REMOVED (master) $ node_modules/.bin/aquifer build
Removing possible existing build...
Executing drush make...
Make new site in the current directory? (y/n):
y
...
Creating symlinks...
REMOVED build created successfully!
jon@Decker ~/projects/REMOVED (master) $ ls -la root
total 16
drwxr-xr-x   4 jon  staff   136 Sep 29 14:29 .
drwxr-xr-x  38 jon  staff  1292 Sep 29 14:02 ..
-rw-r--r--   1 jon  staff     0 Sep 29 12:23 .gitkeep
-rw-r--r--   1 jon  staff  5767 Sep 29 14:28 .htaccess
```

**Comment by fluxsauce** [comment]
Sorry, still happening

**Comment by patrickocoffeyo** [comment]
@fluxsauce I'll re-address this today.. I'm not sure why this would be happening. There's a new PR that restructures this, I'll do some testing once this is merged: https://github.com/aquifer/aquifer/pull/60
